app/tools/request.ts

```diff
@@ -0,0 +1,12 @@
+interface requestArgs {
+  apiPath: string;
```

Comment:
사용 사례를 보니 `'/'`로 항상 시작해야 하는 것 같은데 맞을까요? 맞다면 조금 더 strict하게 이렇게 정의하는 것도 제안해 봅니다!
```suggestion
  apiPath: `/${string}`;
```